modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -784,18 +790,18 @@ TEST(GAPI_Streaming_Types, XChangeScalar)
     fluid_kernels.remove<cv::gapi::core::GMean>();
 
     // FIXME: Now
-    // - fluid kernels take over ocv kernels (including Copy, SubC, & Box3x3)
+    // - fluid kernels take over ocv kernels (including SubC, & Box3x3)
     // - selected kernels (which were removed from the fluid package) remain in OCV
     //   (ConvertTo + some others)
     // FIXME: This is completely awful. User should easily pick up specific kernels
     // to an empty kernel package to craft his own but not do it via exclusion.
     // Need to expose kernel declarations to public headers to enable kernels<..>()
     // on user side.
-    auto kernels = cv::gapi::combine(ocv_kernels, fluid_kernels);
+    auto kernels = cv::gapi::combine(ocv_kernels, fluid_kernels, cv::gapi::streaming::kernels());
```

Comment:
I believe those are available by default. Can you try to remove this?

Reply:
There is `use_only` below

Reply:
I see, thanks